socorro/external/rabbitmq/test_rabbitmq.py

```diff
@@ -1,6 +1,6 @@
 import pika
```

Comment:
it was actually in the ```./scripts``` directory when I found it.  I moved it to the ```.../external/rabbitmq``` directory thinking it a more appropriate location.